source/configuration_manual/dict/index.rst

```diff
@@ -4,8 +4,7 @@
 Dovecot Dictionaries
 ====================
 
-Dovecot's lib-dict can be used to access simple key-value databases. This is
-used by, for example, :ref:`authentication-dict`,
+Dovecot's lib-dict can be used to access simple key-value databases.
 :ref:`lastlogin_plugin`, and :ref:`imap_metadata`. The dictionaries can be
```

Comment:
Shouldn't remove `This is used by, for example,` text since it's followed by more examples than the auth-dict.